thirtyinch-lint/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm"
+    id "jacoco"
+}
+
+repositories {
+    jcenter()
+}
+
+configurations {
+    lintChecks
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+
+    compileOnly "com.android.tools.lint:lint-api:$lintVersion"
+    compileOnly "com.android.tools.lint:lint-checks:$lintVersion"
+
+    testImplementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
```

Comment:
This dependency is already implement as a `implementation` dependency.
Because `testImplementation` extends `implementation` there is no need to declare it again